__std__/nix/plutus/library/plutus-project.nix

```diff
@@ -102,6 +102,8 @@ let
           # > Error: C stack usage  17556409549320 is too close to the limit
           # > Fatal error: unable to initialize the JI
           plutus-core.components.exes.generate-cost-model.buildable = lib.mkForce false;
+
+          # plutus-core.components.benchmarks.cost-model-test.buildable = lib.mkForce false;
```

Comment:
Sadly this is the _only_ thing that fails!
It works in the old CI, not here.
See  https://hydra.iohk.io/eval/1284384

Reply:
Yes, that's it, see here for where we set the platforms: https://github.com/input-output-hk/plutus/blob/master/nix/pkgs/haskell/haskell.nix#L134

I thought the current thing was quite elegant in that it works off the existing nixpkgs `meta` attributes for platforms and works generically, but I won't cry if you bin it.